src/common/args.h

```diff
@@ -66,6 +66,8 @@ enum class OptionsCategory {
     CLI_COMMANDS,
     IPC,
 
+    COMMAND_OPTIONS, // Specific to one or more commands
```

Comment:
is it safe to add in the middle of this enum (e.g. is its index persisted anywhere?)

Reply:
Placement doesn't matter, but I don't think moving closer to COMMANDS, REGISTER_COMMANDS or CLI_COMMANDS really makes sense -- COMMAND_OPTIONS relates to `argsman.AddCommand()` commands (used by bitcoin-wallet and bitcoin-util), which differ from `COMMANDS` (used by bitcoin-tx), `REGISTER_COMMANDS` (also used by bitcoin-tx), and `CLI_COMMANDS` (used by bitcoin-cli for its non-rpc commands, `-generate`, `-netinfo`, `-addrinfo`, `-getinfo`).